tutorials/katacoda/basic-pipeline/init.sh

```diff
@@ -0,0 +1,13 @@
+# Wait for Katacoda to initialize
+sleep 1
+
+# Start Kubernetes
+launch.sh
+
+# # Install Tekton Pipelines
+kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
```

Comment:
I know that the k8s image is on the correct version now. However, if katacoda takes along time to update this image to the latest version of k8s. Is it possible to find ourselves in a situation where the latest version of katacoda is no longer compatible with the latest version of tekton?  Maybe we should we should use the minikube image or a different release `kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.13.2/release.yaml`.

Reply:
my one concern is we'd have to keep it "in sync" and probably set up some _testing_ to make sure we are relatively up-to-date?

I'll change it for now, and for my own sanity set up something to get some validation over time?